sdk/sdktypes/symbol.go

```diff
@@ -18,7 +18,7 @@ var symbolRE = kittehs.Must1(regexp.Compile(`^[a-zA-Z_][\w]*$`))
 
 func (symbolTraits) Validate(s string) error {
 	if s != "" && !symbolRE.MatchString(s) {
-		return errors.New("invalid")
+		return errors.New("illegal symbol (expected [a-zA-Z_][\\w]*)")
```

Comment:
Use raw string:

```go
`illegal symbol (expected [a-zA-Z_][\w]*)`
```

Reply:
I could, but why do we need raw string in this specific case? Is there any difference for the resulting error or I'm missing anything?

Reply:
Raw string lets you use a single `\` in the regular expressions.

Reply:
done